### PR TITLE
Min max improvement

### PR DESCRIPTION
**Pull request by ilyafyodorov**

### CODE REVIEW

File: app/css/main.css [hunk 1]
```diff
@@ -50,6 +50,24 @@ label {
...
+.minVert {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    padding: 0px;
+    margin: 0px;
+}
+
+.maxVert {
```
Comment (STYTZ) [inline app/css/main.css]: Why do you need two same classes?

File: app/js/modules/viewport/minMax/minVert.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+define( function (require) {
```
Comment (STYTZ) [inline app/js/modules/viewport/minMax/minVert.js]: I really think that creating a separate View for such a small thing as min/max label is an overkill. I would do this just within `viewport.js` via `document.createElement`. 
But if you insist on a separate View for it, please have a **single** View, and instantiate it two times for minLabel and maxLabel.

File: app/js/modules/viewport/viewport.js [hunk 3]
```diff
@@ -349,15 +368,81 @@ define(function(require) {
...
+
+                var vertexMin=this.model.attributes.vertexMin;
+                var vertexMax=this.model.attributes.vertexMax;
+                var nodeCoor = this.model.nodes.geometry.attributes.position.array;
+
+                //find arrow start position
+
+                var origMax=new THREE.Vector3(nodeCoor[3*vertexMax], nodeCoor[3*vertexMax+1], nodeCoor[3*vertexMax+2]);
+                var dirMax=origMax.clone();
+                dirMax.sub(this.rotationCenter.position);
```
Comment (STYTZ) [inline app/js/modules/viewport/viewport.js]: Weird behavior when `rotationCenter` coincides with min/maxVertex. Let's use `boundingSphere.center` instead.

Comment (ilyafyodorov) [inline app/js/modules/viewport/viewport.js]: Сhecking of min/max proximity to rotation center was added

**Pull request merged**